BetterThanRottenTomatoAPI/ReviewManagement/ReviewManagement.API/Startup.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using ReviewManagement.Domain.Interfaces;
+using ReviewManagement.Domain.Services;
+using ReviewManagement.Infrastructure.Repositories;
+using ReviewManagement.Infrastructure.Repositories.Interfaces;
+
+namespace ReviewManagement.API
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.AddCors();
```

Comment:
You might need to set up cors properly for it to allow u.i to connect. See how I did it in my service.